Change all true values to boolean in docs (#2330)
jpmorganchase/salt-ds: site/docs/components/button/accessibility.mdx

### Best practices

> - When a Button has both an icon and text, pass `aria-hidden={true}` to the icon so that its text is not announced by screen readers.

Reviewer: ```suggestion
- When a Button has both an icon and text, use`aria-hidden` on the icon so that screen readers do not announce its text.
```
